src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day calendar--month-length">
```

Comment:
Also this class should be,

calendar--start-day-29

With a -29 at the end

so the calendar knows how many days it should go.

Reply:
> Also this class should be,
> 
> calendar--start-day-29
> 
> With a -29 at the end
> 
> so the calendar knows how many days it should go.

I didn't get this one, why shall I add -29 at the end and should there be another calendar--start-day-? like this:
calendar--start-day-wed    calendar--start-day-29